doc/classes/Script.xml

```diff
@@ -43,6 +43,82 @@
 				Returns a dictionary containing constant names and their values.
 			</description>
 		</method>
+		<method name="get_script_documentation_list">
+			<return type="Array" />
+			<description>
+				Returns an array of dictionary containing the documentation of this script and its subclasses.
+				Each dictionary contains the following structure:
+				[codeblock]
+				{
+				    "name": "",
+				    "inherits": "",
+				    "brief_description": "",
```

Comment:
I'm not a fan of enshrining this structure in a (for us) hard to find place we need to keep updated. It also tends to change, so we'd need to maintain it. Maybe we could link to [doc_data.h](https://github.com/godotengine/godot/blob/master/core/doc_data.h#L48)?